pipelines/src/main/java/au/org/ala/pipelines/vocabulary/KeyValuePair.java

```diff
@@ -0,0 +1,34 @@
+package au.org.ala.pipelines.vocabulary;
+
+import java.util.Map;
+
+public class KeyValuePair <K, V> implements Map.Entry<K, V>{
```

Comment:
Does AbstractMap.SimpleImmutableEntry suffice for this ?

https://docs.oracle.com/javase/8/docs/api/java/util/AbstractMap.SimpleImmutableEntry.html

Reply:
removed